src/Wpf.Ui/Controls/Arc/Arc.cs

```diff
@@ -3,27 +3,25 @@
 // Copyright (C) Leszek Pomianowski and WPF UI Contributors.
 // All Rights Reserved.
 
+using System.Windows.Controls;
+using System.Windows.Shapes;
 using Point = System.Windows.Point;
 using Size = System.Windows.Size;
+#pragma warning disable SA1124
+#pragma warning disable CS0108
 
-// ReSharper disable once CheckNamespace
 namespace Wpf.Ui.Controls;
 
-/// <summary>
-/// Control that draws a symmetrical arc with rounded edges.
-/// </summary>
-/// <example>
-/// <code lang="xml">
-/// &lt;ui:Arc
-///     EndAngle="359"
-///     StartAngle="0"
-///     Stroke="{ui:ThemeResource SystemAccentColorSecondaryBrush}"
-///     StrokeThickness="2"
-///     Visibility="Visible" /&gt;
-/// </code>
-/// </example>
-public class Arc : System.Windows.Shapes.Shape
+public class Arc : Shape
 {
+    #region Declarations
```

Comment:
Try to not use regions

Reply:
I can remove them if you want, just tell me. I never understood why anyone shouldn't use them.

Reply:
Purely stylistically, I think they are code smell. They hide part of the code because it is difficult to read, and if the code is difficult to read, it means that it is not good code. It is known that WPF UI needs a lot of work to improve readability and cleanliness, but it is worth working on it